src/cobalt_model/frontmatter.rs

```diff
@@ -394,6 +420,7 @@ impl FrontmatterBuilder {
             is_draft: is_draft.unwrap_or(false),
             collection,
             data,
+            pagination: pagination.and_then(|p| p.build()),
         };
 
         Ok(fm)
```

Comment:
I think this would be a great spot for putting in the feature flag.  If `pagination` is not `None`, then we error.  It shouldn't cause warnings when the flag is not enabled and this function already returns errors.

It could allow someone to configure pagination with `include: None`.  Oh well.

Reply:
I don't know if I'm doing it properly:
```rust
    pagination: pagination.and_then(|p| p.build()),
        }; 

        if !cfg!(feature = "pagination-unstable") && fm.pagination.is_some() {
            Err("Unsupported `pagination` field".into())
        } else {
            Ok(fm)
        }        
```

Reply:
Looks about right.  Running into problems with it?

Reply:
Nope, just first time using `cfg!` :p